### ISSUE TITLE
[A11Y] Android: Unable to linearly navigate through different types of image carousels present on components page

### ISSUE THREAD

**Issue by gkhadke** [ampproject/ampstart#406]
**ENVIRONMENT:**

OS (number): O
Build number: marlin-userdebug O MASTER eng.cbrowe.20170410.155415 dev-keys
Form factor (Smartphone / tablet): PIXEL XL
Browser: Chrome Dev 59.0.3062.4

**PRE-REQUISITE**

1. Talkback is ON

**STEPS TO REPRODUCE:**

1. Open [https://ampstart.com](https://ampstart.com)
2. Linearly Navigate to COMPONENTS link and double tap on it.
3. Try to Linearly navigate through different types of image carousels (e.g. Carousel with text or Carousel with caption) present on the page.

**ACTUAL RESULTS:**

User can move focus on the first visible image and it’s text/caption present under the respective carousels. There are no previous or next buttons. If user tries to swipe linearly further, focus goes on to the next in line element outside carousel which is the “SCRIPT TAG”

Note: Horizontal scroll gesture works as a work around, but it also shows the inconsistency of the carousels from page to page. i.e. Carousel on Homepage have previous and next buttons and they are reachable with linear navigation.

Observation: On desktop, previous and next buttons remains constant. But on On Smartphone or tablet those buttons appears rarely while scrolling the page.

**EXPECTED RESULTS:**

User should be able to navigate through the carousel appropriately using linear navigation.

**ATTACHMENT(S):**

[Carousel with Text.mp4.zip](https://github.com/ampproject/ampstart/files/917626/Carousel.with.Text.mp4.zip)

[Carousel with Caption.mp4.zip](https://github.com/ampproject/ampstart/files/917628/Carousel.with.Caption.mp4.zip)

**Comment by torch2424** [comment]
Unfortuantely, this is a bug with amphtml itself:
https://github.com/ampproject/amphtml/issues/5487

Once the changes come down into PROD, this should be fixed.

cc @camelburrito 

Closing...
